Doc/library/argparse.rst

```diff
@@ -797,6 +797,15 @@ how the command-line arguments should be handled. The supplied actions are:
     >>> parser.parse_args(['--version'])
     PROG 2.0
 
+* ``'extend'`` - This stores a list, and extends each argument value to the
+  list.
+  Example usage::
+
+    >>> parser = argparse.ArgumentParser()
+    >>> parser.add_argument("--foo", action="extend", nargs="+", type=str)
+    >>> parser.parse_args("--foo f1 --foo f2 f3 f4".split())
```

Comment:
I would remove the `split()` call and just write `parser.parse_args("--foo",  "f1", "--foo", "f2", "f3", " f4")`

Reply:
resolved